micro-memcached/src/main/java/com/oath/micro/server/memcached/ElasticachePlugin.java

```diff
@@ -9,6 +9,6 @@
 
     @Override
     public Set<Class> springClasses() {
-            return SetX.of(ConfigureElasticache.class);
+            return SetX.of(ElasticacheConfig.class);
```

Comment:
I guess we will have to add `DistributedCacheFactory.class` also here..

Reply:
updated